index.bs

```diff
@@ -2775,76 +4763,171 @@ partial interface MLGraphBuilder {
 </div>
 
+<details open>
+  <summary>
+    The {{MLGraphBuilder/sigmoid()}} method steps are:
+  </summary>
+  <div algorithm=sigmoid class=algorithm-steps>
+    1. Let |op| be the result of <a>creating an MLActivation</a> given [=this=] and `"sigmoid"`.
+        1. If that [=exception/throws=] an error, re-[=exception/throw=] the error.
+    1. Return |op|.
+  </div>
+</details>
+
 ### The slice() method ### {#api-mlgraphbuilder-slice}
 Produce a slice of the input tensor.
 <script type=idl>
 partial interface MLGraphBuilder {
   MLOperand slice(MLOperand input, sequence<unsigned long> starts, sequence<unsigned long> sizes);
 };
 </script>
-<div algorithm=slice>
+<div>
     **Arguments:**
         - *input*: an {{MLOperand}}. The input tensor.
-        - *starts*: a sequence of {{unsigned long}}. The sequence of unsigned integer values indicating the starting index to slice of each input dimension, of length N where N is the rank of the input tensor. For each dimension *d* of *input*, *starts[d]* indicates the starting index to slice in that dimension. The starting index must be in the range [0, input size - 1] in that dimension.
-        - *sizes*: a sequence of {{unsigned long}}. The sequence of unsigned integer values indicating the number of elements to slice of each input dimension, of length N where N is the rank of the input tensor. For each dimension *d* of *input*, *sizes[d]* indicates the number of elements to slice in that dimension. The size must not be 0 and must satisfy the constraint *starting index + size <= input size* in that dimension.
+        - *starts*: a sequence of {{unsigned long}}. The sequence of unsigned integer values indicating the starting index to slice of each input dimension, of length N where N is the [=rank=] of the input tensor. For each dimension *d* of *input*, *starts[d]* indicates the starting index to slice in that dimension. The starting index must be in the range [0, input size - 1] in that dimension.
+        - *sizes*: a sequence of {{unsigned long}}. The sequence of unsigned integer values indicating the number of elements to slice of each input dimension, of length N where N is the [=rank=] of the input tensor. For each dimension *d* of *input*, *sizes[d]* indicates the number of elements to slice in that dimension. The size must not be 0 and must satisfy the constraint *starting index + size <= input size* in that dimension.
 
     **Returns:** an {{MLOperand}}. The output tensor of the same rank as the input tensor with tensor values stripped to the specified starting and ending indices in each dimension.
 </div>
 
-### The softmax() method ### {#api-mlgraphbuilder-softmax}
+<details open>
+  <summary>
+    The {{MLGraphBuilder/slice(input, starts, sizes)}} steps are:
+  </summary>
+  <div algorithm=slice class=algorithm-steps>
+    1. [=Assert=]: the type of |input| is {{MLOperand}}.
+    1. If |starts| or |sizes| is not a sequence of {{long}}, then [=exception/throw=] a {{TypeError}}.
```

Comment:
Not needed, taken care of by Web IDL. Also, WebIDL calls the type out as `sequence<unsigned long>` - is the signed/unsigned distinction relevant?